### ISSUE TITLE
Jobs+ Iron Miners Hammer not mining 3x3

### ISSUE THREAD

**Issue by Hammander** [DAQEM/JobsPlus-Archive#59]
MC Version: 1.18.2
Jobs+ Version: jobsplus-1.18.2-0.8.2.jar
JEI Version: jei-1.18.2-forge-10.2.1.283.jar
Forge Version: 40.1.92

The Issue I found is:
Unlocked ability to use Iron Miners hammer, was working fine for most of the time, and randomly stopped mining as a 3x3. I tried shifting modes, spawning a new hammer, sneaking and not, and it will not mine more than 1 block at a time.

**Comment by Hammander** [comment]
Update: Logged onto the server again, and the hammer is now working. Unsure what may've caused it to stop.

**Comment by LunafeyLazuli** [comment]
Same problem, discovered that it breaks because there's an NBT being added on the 3x3 hammers that says mode: 0
https://gyazo.com/79ca5bb52b1d44c01ba5d50cd19f2453

**Comment by LunafeyLazuli** [comment]
And this is a freshly made hammer, does it with the shovel too

**Comment by LunafeyLazuli** [comment]
And some people on our same server (cause Hammy and I play on the same server) it only breaks for us, but for everyone else its perfectly fine

**Comment by RageQuitAmy** [comment]
Did you have the vein miner power-up and if so, did you rebind it and even try disabling it?

**Comment by LunafeyLazuli** [comment]
At the time, I did not have vein miner, and I had already set it to `

**Comment by RageQuitAmy** [comment]
Okay, I will try to test it out more. I personally haven't had this problem yet but I don't run servers for that long so that is probably why. There are a few bugs with the miner job that Daqem is aware of and I am sure he will fix them as soon as he is able to.

**Comment by DAQEM** [comment]
I haven't been able to replicate it on 1.19.2 yet but I tweaked some things on the 1.19.2 that should have fixed it. But I haven't had the time to port it to 1.18.2 yet. (mode: 0 means 3x3, mode: 1 is 3x3x3 and so on)

**Comment by LunafeyLazuli** [comment]
Yeah no worries, it's just weird because when it broke for me it would show that NBT tag on the tool, but when my friend would use the same tool that NBT tag would be gone and it worked like normal

**Comment by Yibrian** [comment]
Hello, the same thing happens to me, did you find a solution? Disabling the vein miner power up did not help me
